index.html

```diff
@@ -4,9 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <div class="main-container">
+    <header>
+        <nav class="navbar">
+            <h2 class="logo">Anelisa Didishe</h2>
+            <div id="menu">
+                <i class="fa fa-bars"></i>
+            </div>
+        </nav>
+    </header>
+    <div class="container">
+        <h1>Hey There. I’m Anelisa</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      

```

Comment:
- [ ] Your design should stick as much as possible (e.g. font, colors, images, text, margins) to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Please kindly make sure you have the same fonts (font-size, font-family,...) as the Figma design 😁 🙃